pkgs/racket-doc/scribblings/reference/places.scrbl

```diff
@@ -376,9 +377,9 @@ messages:
  @item{values produced by @racket[shared-flvector],
        @racket[make-shared-flvector], @racket[shared-fxvector],
        @racket[make-shared-fxvector], @racket[shared-bytes], and
-       @racket[make-shared-bytes].}
+       @racket[make-shared-bytes].}]
 
-]}
+@history[#:changed "8.4.0.7" @elem{include @racket[boxes].}]}
```

Comment:
Capitalized "Include" to be consistent with other notes. Also, I think you meant `tech` instead of `racket` here, but I've make it plain "boxes".

Reply:
Looks good - thanks!